src/api/app/assets/stylesheets/webui/timeline.scss

```diff
@@ -8,8 +8,26 @@
         position: relative;
         left: $timeline-offset; // The avatars of users involved in timeline items will be displayed on the timeline border
 
+        .d-inline-flex i:first-child {
+            padding-top: $avatars-counter-size / 2;
+            height: $avatars-counter-size;
+            @extend .bg-white;
+
+            // Align specific icons with the timeline border, this differs depending on the icon shape
```

Comment:
This isn't meant to be pixel-perfect. It's probably 1-2 pixels close to being perfectly aligned, but I don't like magic numbers in Sass files, so this is the closest we'll get.